src/app/tabs/newAdmin/addresses/operations.js

```diff
@@ -37,6 +46,15 @@ export const getChainNameById = (chainId) => {
   return network[0].name;
 };
 
+/**
+ * Returns the index when given the chainId as per slip-0044
+ * @param {chainId} chainId to lookup
+ */
+export const getIndexById = (chainId) => {
+  const network = networks.filter(net => net.id === chainId);
+  return network[0].index;
+};
```

Comment:
I forgot to submit this one:

```suggestion
export const getIndexById = (chainId) => networks.find(net => net.id === chainId).index
```